package-rust.py

```diff
@@ -133,6 +135,8 @@ def run(args):
     rustc_dir + "/COPYRIGHT " + rustc_dir + "/LICENSE-APACHE " + rustc_dir + "/LICENSE-MIT " + \
     "> " + license_file
 run(["sh", "-c", cmd])
+license_rtf = TEMP_DIR + "/LICENSE.rtf"
+run(["pandoc", "-s", license_file, "-o", license_rtf])
```

Comment:
I don't have pandoc on all the build machines (particularly the windows machines). Can invoking pandoc be conditional on using msi for now? I'll probably merge this without turning on msi generation on the bots at first (because it's going to require re-imaging the VMs to contain WIX and pandoc), and if pandoc is invoked unconditionally it'll break the build.